test/clt-tests/comparison-overhead-json-sql/for_fuzzy_search.php

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/php
+<?php
+if (count($argv) < 5) die("Usage: ".__FILE__." <batch size> <concurrency> <docs> <multiplier>\n");
+
+function connectDb() {
+    $host = '127.0.0.1';
+    $port = '9306';
+
+    $m = @mysqli_connect($host, '', '', '', $port);
+    if (mysqli_connect_error()) {
+        die("Cannot connect to Manticore\n");
+    }
+    return $m;
+}
+
+function process($link, $query) {
+    $result = mysqli_query($link, $query);
+    if (!$result) {
+        echo "Ошибка выполнения запроса: " . mysqli_error($link) . "\n";
```

Comment:
English is better. Do we really need two separate scripts when we can use one?

Reply:
- Corrected to English. 
- Clarify what you mean next? If we are talking about two php scripts, then `test-comparison-overhead-json-sql.rec` did not work correctly on my script, or rather its measurements went beyond the specified limits.